src/tracing/index.js

```diff
@@ -39,7 +39,7 @@ function setDefaults() {
 
 
 function shouldEnableTracing() {
-  if (config.tracing && config.tracing.enabled === false) {
+  if (config.tracing && config.tracing.enabled === false && exports.supportedVersion(process.versions.node)) {
     logger.info('Not enabling manual tracing as tracing is not enabled via config.');
```

Comment:
We previously supported manual tracing for all Node.js versions. Why is this now unsupported?

Reply:
With the new CLS, we are now limited by the support provided by cls-hooked.  This is why I proposed also the `engine` entry as a warning on installation.

Reply:
Additionally, falling back to continuation-local-storage as an alternative for 8.0.0 & 8.1.4 won't work as it's broken on those versions.

Reply:
This restriction shouldn't exist. Manual tracing shouldn't have a dependency on automatic tracing.

Reply:
I forgot that this was manual tracing.  I'll remove.